### PR TITLE
Add note that windows binaries are not yet available

### PR DESCRIPTION
**Pull request by bertyhell**
### What does this PR do?

Add context for users wanting to try the native windows bun version. Link to the discord #windows channel

- [x] Documentation
- [ ] Code changes

### How did you verify your code works?

```shell
npm install -g bun                                                                                    0ms
npm ERR! code EBADPLATFORM
npm ERR! notsup Unsupported platform for bun@1.0.0: wanted {"os":"darwin,linux","cpu":"arm64,x64"} (current: {"os":"win32","cpu":"x64"})
npm ERR! notsup Valid os:   darwin,linux
npm ERR! notsup Actual os:  win32
npm ERR! notsup Valid cpu:  arm64,x64
npm ERR! notsup Actual cpu: x64

npm ERR! A complete log of this run can be found in: C:\Users\...-debug-0.log
```

### CODE REVIEW

**Comment by nekoniyah** [comment]
You have to do this on windows: npm install -g bun
You have to install npm before

**Comment by bertyhell** [comment]
@KinyaDev No, its not supported yet. They are working on it. Seems like there were some communication issues with the v1.0 release. The alpha version was supposed to be released by v1.0. But it wasn't since it isn't ready yet

```shell
npm install -g bun                                                                                    0ms
npm ERR! code EBADPLATFORM
npm ERR! notsup Unsupported platform for bun@1.0.0: wanted {"os":"darwin,linux","cpu":"arm64,x64"} (current: {"os":"win32","cpu":"x64"})
npm ERR! notsup Valid os:   darwin,linux
npm ERR! notsup Actual os:  win32
npm ERR! notsup Valid cpu:  arm64,x64
npm ERR! notsup Actual cpu: x64

npm ERR! A complete log of this run can be found in: C:\Users\...-debug-0.log
```

The only way to currently run it on windows is using Windows Subsystem for Linux
https://learn.microsoft.com/en-us/windows/wsl/

**Comment by nekoniyah** [comment]
@bertyhell I know! You have to install it with curl, let me find the link where I found it.
Also, you have to activate WSL (Windows Subsystem Linux) in the Windows features for the bash command to work
`curl -fsSL https://bun.sh/install | bash` and it is on the main page

curl command should work on windows
Ah and you have to install a Linux distribution with like: `wsl --install -d Ubuntu` (I don't know which is the lightweighter)

**Comment by nekoniyah** [comment]
Ah Bun also need unzip to install

**Comment by bertyhell** [comment]
@KinyaDev We're talking about 2 different things here.

1: running bun inside linux inside wsl => this works
2: running bun natively on windows => this does not work

The readme file and this PR are about point 2

**Comment by ShacharHarshuv** [comment]
Any reason this is not merged yet? I found it very confusing in the docs.

File: docs/installation.md [hunk 1]
```diff
@@ -36,6 +36,8 @@ $ proto install bun
 
 ## Windows
 
+Bun binaries for windows are not yet available, but coming soon. Join the [#windows](https://discord.com/channels/876711213126520882/1149339379446325248) discord channel in discord to stay up to date.
+
 Bun provides a _limited, experimental_ native build for Windows. At the moment, only the Bun runtime is supported.
```
Comment (rotu) [inline docs/installation.md]: I'd get rid of the rest of the "Windows" section. Unless there is a link available for the "_limited, experimental_, native build for Windows", there's no point mentioning it.

Comment (bertyhell) [reply docs/installation.md]: @rotu done

File: docs/installation.md [hunk 2]
```diff
@@ -39,7 +39,7 @@ $ proto install bun
 
 ### Windows
 
-Bun provides a _limited, experimental_ native build for Windows. At the moment, only the Bun runtime is supported.
+Bun binaries for windows are not yet available, but coming soon. Join the [#windows](https://discord.com/channels/876711213126520882/1149339379446325248) discord channel in discord to stay up to date.
 
 - `bun <file>`
```
Comment (rotu) [inline docs/installation.md:44]: I think the remainder of this section is still not relevant. The natural question is "okay so binaries for windows are not yet available. But according to this section I can run `bun <file>` - how do I do so?" I think the least confusing thing is to delete lines 44-52.

**Comment by rotu** [comment]
The [1.0 blog post](https://bun.sh/blog/bun-v1.0#bun-more-thing) also still has this misleading sentence:

> For the first time, we're excited to release an experimental, native build of Bun for Windows.

@Jarred-Sumner, could you ~strike~ that sentence please?

**Pull request closed without merging**